cypress.json

```
@@ -3,7 +3,7 @@
   "projectId": "51ef7c",
   "chromeWebSecurity": false,
   "videoUploadOnPasses": false,
-  "defaultCommandTimeout": 15000,
+  "defaultCommandimeout": 15000,
```

should not remain as it was?
```suggestion
  "defaultCommandTimeout": 15000,
```

---

yes it should